code/modules/jobs/job_exp.dm

```diff
@@ -7,12 +7,16 @@ GLOBAL_PROTECT(exp_to_update)
 	set name = "Check Player Playtime"
 	if(!check_rights(R_ADMIN))
 		return
+
 	var/list/msg = list()
 	msg += "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
 	for(var/client/C in GLOB.clients)
 		msg += "<LI> - [key_name_admin(C)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_living() + "</a></LI>"
 	msg += "</UL></BODY></HTML>"
-	src << browse(msg.Join(), "window=Player_playtime_check")
+
+	var/datum/browser/popup = new(mob, "Player_playtime_check", "Player Playtime List")
+	popup.set_content(msg.Join())
+	popup.open()
```

Comment:
Sorry, I'm going to backtrack on this. Admin stuff should be no frills. just use src << browse()